release.sh

```diff
@@ -1,26 +1,21 @@
 #!/bin/bash
 
 printf "version: "
-read version
+read -r version
 
 printf "changelog: "
-read changelog
+read -r changelog
 
 sed -E -i "1 s#^#* $version - $changelog\n#" CHANGELOG
-sed -E -i "s/^version = \"([^\"]*)\"/version = \"$version\"/" lib/Cargo.toml
-sed -E -i "s/^version: \"([^\"]*)\"/version: \"$version\"/" lib/src/bin/.conf/cli.yaml
```

Comment:
I know this file no longer exists, but I would like either @louislang or @andreaphylum to confirm that there aren't other places we need to update the version.

Reply:
Versions were previously being set in the `cli.yaml`. This now gets set by an env var (`CARGO_PKG_VERSION`) at compile time. See: https://github.com/phylum-dev/cli/blob/1dd193bd8ee0dd85fb6027e6ab48d51c8b62efc8/cli/src/app.rs#L17

So we probably need to make sure that env var is set properly.

Reply:
The `CARGO_PKG_VERSION` envvar is set by cargo and exposed when the crate is compiled. It is populated from the value in the project's manifest file...`cli/Cargo.toml` in this case. So...ensuring the manifest is updated to a new version when releasing, like this script does, should be enough.
Ref: ["Environment variables Cargo sets for crates" section in "The Cargo Book"](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates)

Reply:
👍 Figured it was something like that, just wanted to follow the thread there since we ditched the `cli.yaml` a bit ago!